custom-routes-vis/src/main/webapp/script.js

```diff
@@ -12,29 +12,38 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+const LATLNG = {lat: 0.0, lng: 0.0};
+/**
+ * Marker names.
+ * @enum {string}
+ */
+const MarkerNames = {
+  ORIGIN: 'origin',
+  DESTINATION: 'destination',
+};
 // Array holding origin and destination markers.
-let markers = []
+let originDestinationMarkers = []
 
-/** Creates the world map and markers. */
-function createWorldMap() {
+/** Creates the world map with markers. */
+function createWorldMapWithMarkers() {
   map = new google.maps.Map(
     document.getElementById('map'),
     {center: {lat: 38.46049, lng: -5.428423}, zoom: 3});
   
   // Create origin and destination markers, but hides them until user selects
   // coordinates.
-  createMarker('origin-coordinates', 'A', 'Origin',  {lat: 0.0, lng: 0.0});
-  createMarker('destination-coordinates', 'B', 'Destination',
-      {lat: 0.0, lng: 0.0});
+  createMarker('origin-coordinates', 'A', 'Origin', LATLNG);
```

Comment:
Instead of 'Origin', use ORIGIN, i.e. the Enum

Reply:
The general goal is to use thet enum representation.

Reply:
So the problem is that the enum representation is 'origin'(all lowercase). This  has the uppercase 'O'. I'm not super sure what the fix to this would be.

Reply:
Sg